alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/resthandler/RestIndexMonitorAction.kt

```diff
@@ -180,6 +183,27 @@ class RestIndexMonitorAction(
             }
         }
 
+        /**
+         * This function checks whether the [Monitor] has an [HttpInput] with localhost. If so, make sure the port is same as specified in settings.
+         */
+        private fun validateLocalPort(monitor: Monitor, settingsPort: Int) {
+            if (monitor.inputs.isNotEmpty()) {
```

Comment:
No need to have if-else case on checking the input is empty. the for-loop is enough to handle that. Also please fix the indentation.

Reply:
Done.